packages/web-components/src/switch/switch.styles.ts

```diff
@@ -0,0 +1,156 @@
+  spacingVerticalXS,
+} from '../theme/design-tokens.js';
+
+export const styles = css`
+  ${display('inline-flex')}
+
+  :host {
+    align-items: center;
+    flex-direction: row-reverse;
+    outline: none;
+    user-select: none;
+  }
+  :host([label-position='before']) {
+    flex-direction: row;
+  }
+  :host([label-position='above']) {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+  :host([disabled]) .label,
+  :host([readonly]) .label,
+  :host([readonly]) .switch,
+  :host([disabled]) .switch {
+    cursor: not-allowed;
+  }
+  :host .label {
+    position: relative;
+    color: ${colorNeutralForeground1};
+    line-height: ${lineHeightBase300};
+    font-size: ${fontSizeBase300};
+    font-weight: ${fontWeightRegular};
+    font-family: ${fontFamilyBase};
+    padding: ${spacingVerticalXS} ${spacingHorizontalXS};
+    cursor: pointer;
+  }
+  :host([required]) .label:after {
+    content: '*';
+    color: ${colorPaletteRedForeground3};
+  }
+  :host([disabled]) .status-message {
+    color: ${colorNeutralForegroundDisabled};
+  }
+  .switch {
+    display: flex;
+    align-items: center;
+    padding: 0 ${spacingHorizontalXXS};
+    box-sizing: border-box;
+    width: 40px;
+    height: 20px;
+    background-color: ${colorTransparentBackground};
+    border: 1px solid ${colorNeutralStrokeAccessible};
+    border-radius: ${borderRadiusCircular};
+    outline: none;
+    cursor: pointer;
+    margin: ${spacingVerticalS} ${spacingHorizontalS};
+  }
+  .switch:hover {
+    background: none;
+    border-color: ${colorNeutralStrokeAccessibleHover};
+  }
+  .switch:active {
+    border-color: ${colorNeutralStrokeAccessiblePressed};
+  }
+  :host([disabled]) .switch,
+  :host([readonly]) .switch {
+    border: 1px solid ${colorNeutralStrokeDisabled};
+    background-color: none;
+    pointer: default;
+  }
+  .checked-indicator {
+    height: 14px;
+    width: 14px;
+    border-radius: 50%;
+    background-color: ${colorNeutralForeground3};
+    transition-duration: ${durationNormal};
+    transition-timing-function: ${curveEasyEase};
+    transition-property: transform;
+  }
+  :host([aria-checked='true']) .checked-indicator {
+    background-color: ${colorNeutralForegroundInverted};
+    transform: translateX(20px);
+  }
+  :host([aria-checked='true']) .switch:hover .checked-indicator {
+    background: ${colorNeutralForegroundInvertedHover};
+  }
+  :host([aria-checked='true']) .switch:active .checked-indicator {
+    background: ${colorNeutralForegroundInvertedPressed};
+  }
+  .switch:hover .checked-indicator {
+    background-color: ${colorNeutralForeground3Hover};
+  }
+  .switch:active .checked-indicator {
+    background-color: ${colorNeutralForeground3Pressed};
+  }
+  :host([disabled]) .checked-indicator,
+  :host([readonly]) .checked-indicator {
+    background: ${colorNeutralForegroundDisabled};
+  }
+  :host([disabled]) .status-message,
+  :host([readonly]) .status-message {
+    cursor: pointer;
+  }
```

Comment:
This should be removed with the latest version of the template - does the message still exist?